### PR TITLE
Feature/topic 182

### PR DESCRIPTION
**Pull request by MarcoInnovations**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,6 +4,12 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [2.0.0] 2019-04-09
+### Changed
+- Bumped EKS terraform module to 2.3.1
+- Bumped kubernetes version to v1.11
+- Changed DNS readiness probe to accommodate dns namechange to coredns
+
```
Comment (greermichael) [inline CHANGELOG.md]: Add a note that Kubernetes v1.10 is not compatible with these changes

File: VERSION [hunk 2]
```diff
@@ -1 +1 @@
-0.2.3
\ No newline at end of file
+2.0.0
```
Comment (radepal) [inline VERSION]:
```suggestion
0.3.0
```

Comment (MarcoInnovations) [reply VERSION]: We consider the current master branch code as version 1.0.0.  It is stable and functions well.  We will put a v1.0.0 in to mark the current version.  With the change to kubernetes v1.11, there are breaking changes with the previous version.  According to the semantic versioning, this should be a major version increase, hence v 2.0.0.

File: CHANGELOG.md (hunk 1 above)
Comment (MarcoInnovations) [reply CHANGELOG.md]: Added the note as requested.

**Pull request merged**